Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -0,0 +1,1230 @@
+                }
+                else if (handleMoveType == HandleMoveType.Point)
+                {
+                    TransformHandleWithPoint();
+                }
+            }
+        }
+        private void TransformHandleWithRay()
+        {
+            if (currentHandleType != HandleType.none)
+            {
+                currentGrabRay = GetHandleGrabbedRay();
+                Vector3 grabRayPt = currentGrabRay.origin + (currentGrabRay.direction * initialGrabMag);
+
+                if (currentHandleType == HandleType.rotation)
+                {
+                    RotateByHandle(grabRayPt);
+                }
+                else if (currentHandleType == HandleType.scale)
+                {
+                    ScaleByHandle(grabRayPt);
+                }
+            }
+        }
+        private void TransformHandleWithPoint()
+        {
+            if (currentHandleType != HandleType.none)
+            {
+                Vector3 newGrabbedPosition;
+                Vector3 newRemotePoint;
+
+                if (usingPose == false)
+                {
+                    currentPointer.TryGetPointerPosition(out newRemotePoint);
+                    newGrabbedPosition = initialGrabbedPosition + (newRemotePoint - initialGrabPoint);
+                }
+                else
+                {
+                    if (initialGazePoint == Vector3.zero)
+                    {
+                        return;
+                    }
+                    newGrabbedPosition = currentPosePosition;
+                }
+
+                if (currentHandleType == HandleType.rotation)
+                {
+                    RotateByHandle(newGrabbedPosition);
+                }
+                else if (currentHandleType == HandleType.scale)
+                {
+                    ScaleByHandle(newGrabbedPosition);
+                }
+            }
+        }
+
+        private void RotateByHandle(Vector3 newHandlePosition)
+        {
+            Vector3 projPt = Vector3.ProjectOnPlane((newHandlePosition - rigRoot.transform.position).normalized, currentRotationAxis);
+            Quaternion q = Quaternion.FromToRotation((grabbedHandle.transform.position - rigRoot.transform.position).normalized, projPt.normalized);
+            Vector3 axis;
+            float angle;
+            q.ToAngleAxis(out angle, out axis);
+            targetObject.transform.RotateAround(rigRoot.transform.position, axis, angle);
+        }
+        private void ScaleByHandle(Vector3 newHandlePosition)
+        {
+            bool lockBackCorner = false;
+            Vector3 correctedPt = PointToRay(rigRoot.transform.position, grabbedHandle.transform.position, newHandlePosition);
+            Vector3 rigCentroid = rigRoot.transform.position;
+            float startMag = (initialGrabbedPosition - rigCentroid).magnitude;
+            float newMag = (correctedPt - rigCentroid).magnitude;
+
+            if (lockBackCorner == false)
```

Comment:
always false?

Seems like we never get into the [else statement on 421](https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/3044/files#diff-9d24e517f2cbe677a5bbc1a01f74d2efR421)

Reply:
oops this should have been removed.